### PR TITLE
feat: Guides page redesign

### PR DESCRIPTION
**Pull request by iresharma**
## Summary of change
Added changes to match guides page to the new design

## Related issues
- https://github.com/supertokens/main-website/issues/1048

## Checklist
- [ ] Algolia search needs to be updated? (If there is a new sub docs project, then yes)
- [ ] Sitemap needs to be updated? (If there is a new sub docs project, then yes)
- [ ] Checked for broken links? (Run `cd v2 && MODE=production npx docusaurus build`)
- [ ] Changes required to the demo apps corresponding to the docs?

## Remaining TODOs for this PR

### CODE REVIEW

File: v2/src/components/recipeBoxes/guides.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
```
Comment (nkshah2) [inline v2/src/components/recipeBoxes/guides.json:1]: The alignment for the header and the column content is off

Screenshot 2023-03-13 at 6 38 00 PM

File: v2/src/components/recipeBoxes/guides.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  "auth_modes": [
+    {
```
Comment (nkshah2) [inline v2/src/components/recipeBoxes/guides.json:3]: The arrow icon should rotate in position. Currently it looks like the image is being swapped and the baseline for the two images is different

https://user-images.githubusercontent.com/18233774/224711392-b1f8255c-8ed0-45e6-bdc0-67ffdca111bd.mov

File: v2/src/components/recipeBoxes/guides.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      "title": "Email password Login",
+      "icon": "emailPass",
```
Comment (nkshah2) [inline v2/src/components/recipeBoxes/guides.json:5]: The spacing to the right of the select is wrong. Also the designs have a minimum width for the select it seems

Screenshot 2023-03-13 at 6 43 05 PM

File: v2/src/components/recipeBoxes/guides.json [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+      "path": "/docs/emailpassword/introduction",
+      "img": "emailPassSS"
```
Comment (nkshah2) [inline v2/src/components/recipeBoxes/guides.json:7]: The icons used for the recipes and addons etc are not as per designs

Screenshot 2023-03-13 at 6 44 41 PM
Screenshot 2023-03-13 at 6 44 38 PM
Screenshot 2023-03-13 at 6 44 18 PM

File: v2/src/components/recipeBoxes/guides.json [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    },
+    {
```
Comment (nkshah2) [inline v2/src/components/recipeBoxes/guides.json:9]: The ordering of addons, recipes etc is not as per designs

File: v2/src/components/recipeBoxes/guides.json [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+      "title": "Social Login",
+      "icon": "social",
```
Comment (nkshah2) [inline v2/src/components/recipeBoxes/guides.json:11]: Spacing around the images is not as per design

Screenshot 2023-03-13 at 6 48 12 PM

File: v2/src/components/recipeBoxes/guides.json [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+      "path": "/docs/thirdparty/introduction",
+      "img": "socialSS"
```
Comment (nkshah2) [inline v2/src/components/recipeBoxes/guides.json:13]: The alignment inside the card is consistent in the designs but different in the output

## Designs
Screenshot 2023-03-13 at 6 50 26 PM

## Output
Screenshot 2023-03-13 at 6 50 19 PM

File: v2/src/components/recipeBoxes/guides.json [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+    },
+    {
```
Comment (nkshah2) [inline v2/src/components/recipeBoxes/guides.json:15]: The icon size is not as per designs

Screenshot 2023-03-13 at 6 52 56 PM

File: v2/src/components/recipeBoxes/guides.json [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+      "title": "Email password + Social Login",
+      "icon": "emailPassPlusSocial",
```
Comment (nkshah2) [inline v2/src/components/recipeBoxes/guides.json:17]: The inner shadow in the hover state is not as per designs

Screenshot 2023-03-13 at 6 58 57 PM

File: v2/src/components/recipeBoxes/guides.json [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+      "path": "/docs/thirdpartyemailpassword/introduction",
+      "img": "emailSocialSS"
```
Comment (nkshah2) [inline v2/src/components/recipeBoxes/guides.json:19]: All images and icons seem blurry, make sure to use 2x or 3x assets

**Review by nkshah2 — CHANGES_REQUESTED** [review]

File: v2/src/components/recipeBoxes/guides.json (hunk 4 above)
Comment (iresharma) [reply v2/src/components/recipeBoxes/guides.json:7]: Didn't get this point, the icons are downloaded from the figma file

Comment (nkshah2) [reply v2/src/components/recipeBoxes/guides.json]: Screenshot 2023-03-15 at 12 51 35 PM
Screenshot 2023-03-15 at 12 51 29 PM
Screenshot 2023-03-15 at 12 51 16 PM
Screenshot 2023-03-15 at 12 51 11 PM

File: v2/src/components/recipeBoxes/guides.json (hunk 2 above)
Comment (nkshah2) [reply v2/src/components/recipeBoxes/guides.json:3]: This is still pending

File: v2/src/components/recipeBoxes/guides.json (hunk 9 above)
Comment (nkshah2) [reply v2/src/components/recipeBoxes/guides.json:5]: This is still pending

File: v2/src/components/recipeBoxes/guides.json (hunk 10 above)
Comment (nkshah2) [reply v2/src/components/recipeBoxes/guides.json:7]: This is still a problem

File: v2/src/components/recipeBoxes/guides.json [hunk 11]
```diff
@@ -0,0 +1,79 @@
+{
+  "auth_modes": [
```
Comment (nkshah2) [inline v2/src/components/recipeBoxes/guides.json:2]: For long recipe names the spacing is off

Screenshot 2023-03-15 at 12 54 11 PM

File: v2/src/components/recipeBoxes/guides.json [hunk 12]
```diff
@@ -0,0 +1,79 @@
...
+    {
+      "title": "Email password + Social Login",
```
Comment (nkshah2) [inline v2/src/components/recipeBoxes/guides.json:4]: The hover logic does not account for the outer container it seems


https://user-images.githubusercontent.com/18233774/225236745-634aa403-31a9-4984-82ae-4c0595e1bb9b.mov

**Review by nkshah2 — CHANGES_REQUESTED** [review]

File: v2/guides/guides.mdx [hunk 13]
```diff
@@ -1,20 +1,24 @@
 ---
 id: guides
-title: Recipe guides
+title: User Recipes
```
Comment (nkshah2) [inline v2/guides/guides.mdx:3]: When you toggle the select it defaults to "Select Recipe" instead of keeping its value

**Review by nkshah2 — CHANGES_REQUESTED** [review]

**Review by nkshah2 — APPROVED** [review]

**Pull request merged**